exercises/07-test_template.md

```diff
@@ -47,25 +47,32 @@ Verify That Error Page Is Visible
 So, basically what most of us did was copy-paste the first test several times, change the test names
 and change the entered usernames and passwords. The tests do test different things and work well, but it
 doesn't really make sense to copy-paste several lines and modify 2 parameters. Instead, we should use
-test templates: http://robotframework.org/robotframework/latest/RobotFrameworkUserGuide.html#test-templates
+[test templates](http://robotframework.org/robotframework/latest/RobotFrameworkUserGuide.html#test-templates).
 
-So far we've been doing so-called workflow tests. Test templates are so-called data driven tests, which are
+So far we've been doing so-called workflow tests. Test templates are so-called **data driven tests**, which are
 applicable when you need to run the same test, multiple times, every time with a couple of slightly altered arguments.
 
-Let's get started. Rename the first test case to `Template Error Page Is Visible When Using Incorrect Credentials`
+## Exercise
+
+- Rename the first test case to `Template Error Page Is Visible When Using Incorrect Credentials`
 and move it from the `Test Cases` table under the `Keywords` table.
 
 The `Template Error Page Is Visible When Using Incorrect Credentials` keyword starts with a
```

Comment:
Why have we changed the test case name to include `Template` in the name?

Reply:
It has not been "changed". It has been like this since the "beginning of time" (see for example the [rf-katas-sl](https://github.com/eficode-academy/rf-katas-sl/blob/master/setup/07-test_template_rule.py#L6) verification for this exercise).

Reply:
If you read the exercise through, we first change the first test case name (before we make it a test template), then we move it under `keywords`, and finally we add the  `Test Template` to `settings`.

Reply:
I don't know at which point someone has changed it to this, but I think it's a bad pattern, as in general the keyword can be used in other test cases not as a template.